packages/generators/app/lib/utils/check-requirements.js

```diff
@@ -1,13 +1,21 @@
 'use strict';
 
+const { red, green, bold } = require('chalk');
+const semver = require('semver');
+
 module.exports = function checkBeforeInstall() {
-  var currentNodeVersion = process.versions.node;
-  var semver = currentNodeVersion.split('.');
-  var major = semver[0];
+  const currentNodeVersion = process.versions.node;
+  const minNodeVersion = '14.19.1'; // greater than or equal to this
```

Comment:
Would it make sense to read that information from the package.json, so that we do not duplicate the information?

Reply:
Yes, it would be ideal to use the value from the generated package.json, but the way it works right now we would need to either
a) parse it from the engines.node string (which is a different format than semver uses), which could potentially cause problems in the future anyway if we update that string to a different format
or
b) rewrite both this and the package.json generator to use a singular source of supported version

Either way adds too much complexity in my opinion, especially since at this point that it should only be a warning anyway rather than a `process.exit`

Reply:
I'm think the engines value in package.json is semver compatible you should try it with the semver package to make sure :)

Reply:
You're right, it is! Their documentation made it sound like it required logical operators between versions.

As soon as I have some time, I'll have it pull the value from the generated package.json and compare against that, and make it an error again.